dev/NavigationView/NavigationViewItem.cpp

```diff
@@ -144,9 +144,16 @@ void NavigationViewItem::UpdateRepeaterItemsSource()
             }
             return MenuItems().as<winrt::IInspectable>();
         }();
+        m_itemsSourceViewChanged.revoke();
```

Comment:
I think you need to revoke this listener in `UnhookEventsAndClearFields` too:

https://github.com/microsoft/microsoft-ui-xaml/blob/50577c70b5f0879a6e81e3fa402f41364c84ee90/dev/NavigationView/NavigationViewItem.cpp#L918-L928

Reply:
Do I also need to rename the variable into `m_itemsSourceViewCollectionChangedRevoker` to be inline with the other revokers?

Reply:
Good point, yes that would be great!

Reply:
Just pushed a commit doing this.